AlayaCare Cloud Tech Radar.csv

```diff
@@ -14,10 +14,15 @@
 "Adopt","Vue CLI","languages-and-frameworks","FALSE","<p><a href='https://cli.vuejs.org/'>Vue CLI</a> offers standard tooling for Vue.js development.</p>"
 "Adopt","Vue Test Utils","languages-and-frameworks","FALSE","<p><a href='https://vue-test-utils.vuejs.org/'>Vue Test Utils</a> is the official unit testing utility library for Vue.js.</p>"
 "Adopt","Webpack","languages-and-frameworks","FALSE","<p><a href='https://webpack.js.org/'>Webpack</a></p>"
+"Adopt","Snowflake","Platforms","<p></p>"
```

Comment:
You're missing the `isNew` column, let set it to FALSE for now.
Also add a link to the platform website when available.

```suggestion
"Adopt","Snowflake","Platforms","FALSE","<p><a href='https://www.snowflake.com/'>snowflake.com</a></p>"
```